05_stringer/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var testMatrix = []struct {
+	input  IPAddr
+	output string
```

Comment:
I'd call this `expected` or `expectedOutput`

Reply:
fixed this